Source/WebCore/rendering/TextDecorationPainter.cpp

```diff
@@ -255,10 +255,22 @@ void TextDecorationPainter::paintBackgroundDecorations(const TextRun& textRun, c
     // These decorations should match the visual overflows computed in visualOverflowForDecorations().
     auto underlineRect = FloatRect { boxOrigin, FloatSize { decorationGeometry.textBoxWidth, decorationGeometry.textDecorationThickness } };
     auto overlineRect = underlineRect;
-    if (decorationType.contains(TextDecorationLine::Underline))
-        underlineRect.move(0.f, decorationGeometry.underlineOffset);
-    if (decorationType.contains(TextDecorationLine::Overline))
-        overlineRect.move(0.f, decorationGeometry.overlineOffset);
+
+
+    // For now lets ignore the TextDecorationLine if we have some TextUnderlinePosition. This is wrong, but let's go with it.
```

Comment:
I don't know what that means. Could you please explain why? (e.g. put a FIXME with a bug ID with some explanation)